src/keystore/gosbot.rs

```diff
@@ -0,0 +1,96 @@
+//! Read and write a go-sbot secret (identity) file.
+
+#![warn(missing_docs)]
+
+use async_std::{
+    io::{Read, Write},
+    prelude::*,
+};
+
+use std::string::ToString;
+
+use super::{
+    error::{Error, Result},
+    JsonSSBSecret, OwnedIdentity, CURVE_ED25519,
+};
+use crate::crypto::{ToSodiumObject, ToSsbId};
+
+fn to_io_error<T: ToString>(err: T) -> async_std::io::Error {
```

Comment:
`to_io_error` function also exists in https://github.com/Kuska-ssb/ssb/blob/master/src/keystore/patchwork.rs#L25, what about creating a new file `src/keystore/util.rs` and put this function there?

Reply:
Good idea. Done.